src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -1029,11 +1029,11 @@ void NetworkOPsImp::apply (std::unique_lock<std::mutex>& batchLock)
     batchLock.unlock();
 
     {
-        auto lock = make_lock(app_.getMasterMutex());
+        auto MasterLock = make_lock(app_.getMasterMutex(), std::defer_lock);
         bool changed = false;
         {
-            std::lock_guard <std::recursive_mutex> lock (
-                m_ledgerMaster.peekMutex());
+            auto LedgerLock = make_lock(m_ledgerMaster.peekMutex(), std::defer_lock);
```

Comment:
Ultra-minor nit... variable names typically have an initial lower case letter.  So I'd expect `MasterLock` to be `masterLock` and `LedgerLock` to be `ledgerLock`.